src/main/java/no/entur/antu/flex/validation/config/TimetableDataFlexValidatorConfig.java

```diff
@@ -61,18 +75,58 @@ public NetexIdValidator flexNetexIdValidator(@Qualifier("validationReportEntryFa
     }
 
     @Bean
-    public NetexValidatorsRunner flexTimetableDataValidatorsRunner(NetexSchemaValidator netexSchemaValidator,
-                                                                   @Qualifier("flexTimetableDataXPathValidator") XPathValidator flexXPathValidator,
+    public FileNameValidator fileNameValidator(@Qualifier("validationReportEntryFactory")
+                                               ValidationReportEntryFactory validationReportEntryFactory) {
+        return new FileNameValidator(validationReportEntryFactory);
+    }
+
+    @Bean
+    public NetexValidatorsRunner flexTimetableDataValidatorsRunner(@Qualifier("flexTimetableDataXPathValidator") XPathValidator flexXPathValidator,
                                                                    @Qualifier("flexNetexIdValidator") NetexIdValidator netexIdValidator,
+                                                                   @Qualifier("netexIdUniquenessValidator") NetexIdUniquenessValidator netexIdUniquenessValidator,
+                                                                   NetexSchemaValidator netexSchemaValidator,
                                                                    VersionOnLocalNetexIdValidator versionOnLocalNetexIdValidator,
                                                                    VersionOnRefToLocalNetexIdValidator versionOnRefToLocalNetexIdValidator,
                                                                    ReferenceToValidEntityTypeValidator referenceToValidEntityTypeValidator,
                                                                    NetexReferenceValidator netexReferenceValidator,
-                                                                   @Qualifier("netexIdUniquenessValidator") NetexIdUniquenessValidator netexIdUniquenessValidator) {
-        List<NetexValidator> netexValidators = List.of(flexXPathValidator, netexIdValidator, versionOnLocalNetexIdValidator, versionOnRefToLocalNetexIdValidator, referenceToValidEntityTypeValidator, netexReferenceValidator, netexIdUniquenessValidator);
+                                                                   FileNameValidator fileNameValidator) {
+        List<NetexValidator> netexValidators = List.of(
+                fileNameValidator,
+                flexXPathValidator,
+                netexIdValidator,
+                versionOnLocalNetexIdValidator,
+                versionOnRefToLocalNetexIdValidator,
+                referenceToValidEntityTypeValidator,
+                netexReferenceValidator,
+                netexIdUniquenessValidator
+        );
         // do not ignore SiteFrame
         NetexXMLParser netexXMLParser = new NetexXMLParser(Set.of());
         return new NetexValidatorsRunner(netexXMLParser, netexSchemaValidator, netexValidators);
     }
 
+    @Bean
+    public NetexValidatorsRunner importFlexTimetableDataValidatorsRunner(@Qualifier("importFlexTimetableDataXPathValidator") XPathValidator flexXPathValidator,
```

Comment:
javadoc comments could be used here to explain the difference between the 2 validation profiles